pyproject.toml

```diff
@@ -45,6 +45,7 @@ dependencies = [
     "anyio",
     "asyncer==0.0.8",
     "cachetools",
+    "transformers>=4.47.0",
```

Comment:
Thanks @DigitalPig for the PR!! Why add transformers?

Reply:
```
ImportError while importing test module '/home/digitalpig/Codes/dspy/tests/modules/test_hf_model.py'.
Hint: make sure your test modules/packages have valid Python names.
Traceback:
/home/linuxbrew/.linuxbrew/opt/python@3.12/lib/python3.12/importlib/__init__.py:90: in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
tests/modules/test_hf_model.py:2: in <module>
    from transformers import AutoModelForSeq2SeqLM
E   ModuleNotFoundError: No module named 'transformers'
```

Maybe the best place to put is the [dev] section?